### DISCUSSION TITLE
Google search console veryfication

### DISCUSSION THREAD

**Discussion by Glakuta** [vercel/community#2698] [Help]
### Summary

Hi! I have an issue with my domain that is redirecting to vercel app. The domain is redirecting properly but I cant veryfiy in google search console. I've add txt record to dns but google is throwing error.

### Example

https://www.assistance24-krakow.pl/

### Steps to Reproduce

In google browser paste site:assistance24-krakow.pl

**Comment by amyegan** [comment]
Hi @Glakuta. This sounds similar to https://github.com/vercel/community/discussions/2611. If that discussion seems to match what you're seeing, you can find ongoing updates in this thread: https://github.com/vercel/community/discussions/2611#discussioncomment-6047511
